pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -222,7 +222,7 @@ func TestReconcile(t *testing.T) {
 					},
 				},
 			},
-			expectedErr:             false,
+			expectedErr:             true,
```

Comment:
I debugged this case locally, the output error is "kubeconfig is missing", should we consider to change the token in Line 220 to kubeconfig to make this case pass?

Reply:
A positive case will be added in a separated PR.